app.js

```diff
@@ -48,10 +53,10 @@ document.getElementById('books').addEventListener('click', (event) => {
   const isButton = event.target.nodeName === 'BUTTON';
   if (!isButton) return;
   BooksStoreUtilities.removeByTitle(event.target.value);
```

Comment:
- [ ] There is a bug in your app. When I add duplicate books and remove one, both books are cleared from `localStorage` and the page. Kindly fix this.

  **SUGGESTION:** You can implement your deletion using a kind of unique `id`, instead of `title`. This may require adding an `id` property when creating your book, and deleting using that `id`.

  Duplicate books are added | One book is deleted, but one was clicked
  :---:|:---:
  ![jenad-1](https://user-images.githubusercontent.com/36542986/159664327-d9687640-d95c-4d38-b087-d3bce7376a67.jpg) | ![jenad-2](https://user-images.githubusercontent.com/36542986/159664339-034a7426-dd7e-45c6-86af-4eb04216aff9.jpg)